tests/test_rooms/test_create_room.py

```diff
@@ -29,6 +28,8 @@ def test_room_creation(self):
 
         query = self.app_test.post(query_string, headers=headers)
         expected_response = query_string_response
+        print(query.data)
+        print(expected_response)
 
```

Comment:
You can take these print statements out. They are not needed.

Reply:
Thanks on it @kampkelly